tests/tests/control_2threads_p7

```diff
@@ -111,8 +111,9 @@ export DO_CA=.true.
 export CA_SGS=.true.
 export CA_GLOBAL=.false.
 export NCA=1
-export SCELLS=2600
-export TLIVES=1800
+export NCELLS=5
+export NLIVES=12
+export NTHRESH=18
```

Comment:
I am wondering if the control_p7 related tests need the default values of NCELLS/NLIVES/NTHRESH, can we define those variables in the export_fv3 section in the default_vars.sh?

Reply:
My regression tests ran and passed (with first -c and then -m) without adding these in default_vars.sh, do you still want me to add them here?

Reply:
I do think it would be cleaner to have them specified in the export_fv3 section for the control*p7 tests instead of each individual test. But since you haven't tested it that way, I think we could add that bit of clean-up in a later PR which does not change baselines.

Reply:
Ok, yes. I'm happy to help with that if needed.